rust/phoenix-sdk/src/sdk_client.rs

```diff
@@ -192,7 +193,10 @@ impl SDKClient {
         let rt = tokio::runtime::Runtime::new()?;
         rt.block_on(Self::new_with_market_keys(market_keys, payer, url))
     }
+}
 
+/// Mutable functions that modify the internal state of the SDKClient
```

Comment:
I like these new comments that clarify the purposes of the different `impl` blocks